draft-ietf-intarea-provisioning-domains.md

```diff
@@ -332,11 +333,13 @@ In other words, different Explicit PvDs MAY be advertised with RAs using
 the same link-local source address; but different Implicit PvDs, advertised
 by different RAs, MUST use different link-local addresses because
 these Implicit PvDs are identified by the source addresses of the
-RAs.
+RAs. If a link-local address on the router is changed, then any new RA
```

Comment:
Not all changes of link-local addresses would have the described consequences. I tried to rephrase, but am not super happy with the result either. Here is a proposed text:
> In particular, if the source link-local address used when sending such RAs is changed, the RA will be interpreted as a new Implicit PvD by PvD-aware hosts.

Reply:
Not too happy with both text ;-) What about

> If a link-local address on the router is changed, then any new RA If a link-local address on the router is changed, then any new RA will be interpreted as a different Implicit PvD by PvD-aware hosts.

Even with a PvD option, the outside options are implicit PvD and 'different' because I may have existed 1 hour before (no need to create)